examples/basic/autocomplete.py

```diff
@@ -11,14 +11,15 @@
 
 async def autocomplete_response(
     ctx: crescent.AutocompleteContext, option: hikari.AutocompleteInteractionOption
-) -> Sequence[hikari.CommandChoice]:
+) -> Sequence[Tuple[str, str]]:
```

Comment:
Could you use lowercase `tuple` here?

Reply:
I could, but Tuple was already imported and used elsewhere, and to use tuple I'd need to add `from __future__ import annotations`. Up to you though

Reply:
i'd prefer if you added the annotations import i think

Reply:
Alright I did that - and since the return types should be more narrow, I changed Sequence to list. I can revert that part though